README.md

```diff
@@ -76,7 +76,7 @@ Other release could be downloaded from [here][release-all].
 
 # Installation
 
-Please check [here][docs-user-getting-started] for detailed instruction on how to install and setup up Strongbox on the 
```

Comment:
@mohit0920 : If I were to change anything here, it would be to correct "instruction" to "instructions" (plural). If you'd like to change that, it would be great, otherwise I would rather dimiss this particular change ("on" vs. "about").